util/fipstools/test_fips.c

```diff
@@ -214,12 +214,12 @@ int main(int argc, char **argv) {
     goto err;
   }
   printf("About to AES-ECB decrypt ");
-  hexdump(output, out_len);
+  hexdump(output, sizeof(kPlaintext));
   for (size_t j = 0; j < out_len / 16; j++) {
```

Comment:
should this be `out_len`?

Reply:
`out_len` was being abused in a few places across multiple algorithms and was confusing to track. So I intentionally started to strip it out, but I think what is missing here is that the for loop below this change should be fixed.